File: CMS-API/DTOs/StockDetailByMaterialNo.cs
namespace API.DTOs
{
    public class StockDetailByMaterialNo
    {

        public string Kind { get; set; }
        public string MateriaNo { get; set; }
        public string OrderStage { get; set; }
        public string Article { get; set; }
        public string ModelNo { get; set; }

        public string StockNo { get; set; }
        public string Location { get; set; }
        public string MaterialQty { get; set; }
        public string TestResult { get; set; }
        public string InBoxDate { get; set; }

    }
}

File: MMS-SPA/src/app/core/_models/stock-detail-by-material-no.ts
import { ModelInterface } from "./interface/model-interface";

export class StockDetailByMaterialNo implements ModelInterface{
    partNo :string;
    kind :string;
    materiaNo :string;
    orderStage :string;
    article :string;
    modelNo :string;

    stockNo :string;
    location :string;
    materialQty:string;
    testResult :string;
    inBoxDate :string;
}
